src/pages/[platform]/deploy-and-host/sandbox-environments/setup/index.mdx

```diff
@@ -31,11 +31,15 @@ export function getStaticProps(context) {
 
 You can use a personal cloud sandbox environment that provides an isolated development space to rapidly build, test, and iterate on a fullstack app. Each developer on your team can use their own disposable sandbox environment connected to cloud resources.
 
-<Callout>
 
-Cloud sandbox environments are not intended for production workloads.
+**Cloud sandbox environments** are designed for development purposes and are not intended for production workloads. To accelerate deployments, Amplify utilizes **CDK hot swapping** where supported, enabling rapid updates to resources such as Lambda functions and AppSync resolver templates without requiring a full redeployment.  
```

Comment:
```suggestion
**Cloud sandbox environments** are designed for development purposes and are not intended for production workloads. To accelerate deployments, Amplify utilizes **CDK hot swapping** where supported, enabling rapid updates to resources such as Lambda functions and AppSync resolver templates without requiring a full redeployment.  
```

few nits for service names since they're the first reference to the service in the doc
- Lambda -> AWS Lambda
- AppSync -> AWS AppSync

(and similar for ddb)